addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -552,9 +552,12 @@
     <div class="oe_slide_js_embed_code_widget mt-4">
         <h5 class="mt0">Embed in your website</h5>
         <div class="form-group">
-            <textarea class="form-control slide_embed_code" readonly="readonly" onClick="this.select();"><t t-esc="slide.embed_code"/></textarea>
+            <textarea class="form-control slide_embed_code" readonly="readonly" onClick="this.select();">
+                <!-- Use the external embedding URL here, see python controller '_slide_embed' method for more details. -->
+                <t t-esc="slide.embed_code.replace('/embed/', '/embed_external/')"/>
```

Comment:
Mmmh ... maybe we could have a with_context and recompute the field based on context key ? That way we don't have to handle URLs manually in a template and let the code do it neatly. Normally this should not impact performances anymore since cache is not destroyed just for context update ^^ .

Reply:
Well if people updates the view and mess with this code ... I think it is better to avoid some code in views, even simple like that.

Counter-attack: why not ``embed_code_external`` field then ? Anyway those are computed, that was we have one field per embed type, and computed in the same method anyway.